### PR TITLE
Add Profile settings and CSS

### PR DESCRIPTION
**Pull request by Lenkly**
Hi to the two of you,

I have two little issues with the commit I couldn't solve myself. 
First one: justify-content sets the BENUTZERNAME always to the end of the line, otherwise I have to align them in the center which doesn't look good either. 
Second: I can't stretch the dark background to 100% width.

Hope anything else works.

PS: I can't change anything on the switch-sliders, as even renaming the classes make them act like a little bitch 🤪

### CODE REVIEW

File: src/components/_profile.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.profile__personal {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  padding-top: 0.8em;
+  padding-bottom: 0.8rem;
+}
```
Comment (hauptdigital) [inline src/components/_profile.scss]: if you use `display:flex;` you can also use `flex-direction: row; ` (which is the default) or `flex-direction: column; `. Maybe this helps.

File: src/components/_profile.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
+@use '../base.scss';
+
+.profile--dark {
+  background-color: base.$secondary-color;
+  color: base.$font-white;
+  line-height: 1.7em;
+  padding-bottom: 0.8rem;
+  padding-top: 0.8rem;
+}
```
Comment (hauptdigital) [inline src/components/_profile.scss]: Please add these lines to stretch the profile:

`margin: 0 -15px;`
`padding-left: 15px;`
`padding-right: 15px;`

**Comment by hauptdigital** [comment]
Profile looks good. Switch sliders can stay like this :-)

File: src/components/_profile.scss (hunk 1 above)
Comment (Lenkly) [reply src/components/_profile.scss]: Didn't work like I intended it to do, but looks even better now 😃

**Pull request merged**